Userland/Shell/PosixParser.cpp

```diff
@@ -1437,15 +1458,20 @@ ErrorOr<RefPtr<AST::Node>> Parser::parse_for_clause()
         Optional<AST::Position> {});
 }
 
-RefPtr<AST::Node> Parser::parse_word_list()
+RefPtr<AST::Node> Parser::parse_word_list(bool allow_newlines)
```

Comment:
This would be easier to read in parse_bash_like_list if you used a bool-like enum `AllowNewlines::{Yes, No}`